### ISSUE TITLE
Changing the expiry to 10 minutes results in "expires timeDiffFutureMinutes = in 9 minutes" in the expiry field

### ISSUE THREAD

**Issue by catlee** [mozilla-services/screenshots#4867]
STR:
1. Take screenshot, upload to screenshots.firefox.com
2. Click on "expires in 14 days"
3. Change to something like 10 minutes

Result:
https://screenshots.firefox.com/poRUQScezvpCbKKl/screenshots.firefox.com

**Comment by SoftVision-CosminMuntean** [comment]
I have test this issue on latest Firefox 62.0 release and latest Nightly 64.0a1 build and I have managed to reproduce it.

It seems that it is related to localization. I have managed to reproduce this if I change the preferred language from "about:preferences#general" page in English (Australia) [en-au] or English (Canada) [en-ca]. 

The issue is reproducible for the "10 minutes", "1 hour" and "1 day" options.

**Comment by chenba** [comment]
This bug appears to have gone away.

**Comment by catlee** [comment]
It's still happening for me:
https://screenshots.firefox.com/4uBLnx9pHLIaCxxQ/mail.google.com

**Comment by chenba** [comment]
@catlee Sorry I forgot to change my language pref earlier.

**Comment by chenba** [comment]
Some `.ftl`s contain syntax errors.  Here's what `en-CA` looks like, for example:

```
timeDiffFutureMinutes =
    timeDiffFutureMinutes = { $number ->
        [one] in 1 minute
       *[other] in { $number } minutes
    }
timeDiffFutureHours =
    timeDiffFutureHours = { $number ->
        [one] in 1 hour
       *[other] in { $number } hours
    }
```

**Comment by SoftVision-CosminMuntean** [comment]
Unfortunately, I cannot verify this issue since on the latest Screenshots dev version the FxA feature is merged and we no longer have the option to select the expiry date for the shots.

Is there another way to test this? Or we should wait until the fix will be deployed on the stage server?

**Comment by chenba** [comment]
@SoftVision-CosminMuntean Do you always test with a clean profile?  If not, and you have some older shots, then you could check those.  Any shot that's expiring within a day will have that issue.  Otherwise, there's no quick way of testing this on dev or stage.

Note that there's no code change in the fix; only translation files were updated.

**Comment by SoftVision-CosminMuntean** [comment]
Usually, I test with a clean profile. Unfortunately, I had a lot of profiles and I deleted most of them and now I can't find an old profile with shots saved.

But, I had an idea. I have changed the preferred language to "en-ca" language and saved a shot. Then, I have changed the system date to be 13 days in the future and  I have navigated to the saved shot page and the "expires in 23 hours" message was correctly displayed.

However, just to be sure I have done this with two different profiles. On one of them, I have the Screenshots dev version installed and on the other the production Screenshots version.
See this screenshot:
![23 h](https://user-images.githubusercontent.com/19226534/45556990-43876100-b845-11e8-8471-379d618f8cf3.png)

I'm not sure if this is the right way to verify this, but I will re-check after the changes will be deployed in production to be sure.
